extensions/mongoProcessor/MongoQueueProcessor.js

```diff
@@ -122,6 +125,13 @@ class MongoQueueProcessor {
                               { error: sourceEntry.error });
             return process.nextTick(() => done(errors.InternalError));
         }
+        // TODO-FIX:
+        // Depends on the filter data. Need a way of determining the
+        // zenko bucket.
+        // if entry is for another site, simply skip/ignore
+        if (this.site !== kafkaEntry.bucket) {
```

Comment:
to support site-level consumers, need to ignore entries for another site. This may need some revision based on kafka entries produced

Reply:
Ok, this is inline with what I mentioned yesterday. We need to move the logic of processing of an entry to be ingested from producer to consumer.